conf/nginx/nginx.conf

```diff
@@ -62,7 +62,10 @@ http {
         # Not to be confused with NGINX's builtin $proxy_host variable!
         #
         # ?proxy_path: The path part of the URL-to-be-proxied, e.g. "/bar/gar.pdf"
-        location ~ ^/proxy/static/(?<sec>[a-zA-Z0-9-_]+)/(?<exp>\d+)/(?<proxy_scheme>https?://)(?<proxy_hostname>[^/]+)(?<proxy_path>.*) {
+
+        location ~ ^/proxy/static/(?<sec>[a-zA-Z0-9-_]+)/(?<exp>\d+)/(?<proxy_scheme>https?://?)(?<proxy_hostname>[^/]+)(?<proxy_path>.*) {
+            rewrite ^/proxy/static/[^/]+/[^/]+/(https?):/([^/].*)$ $1://$2 break;
+
```

Comment:
Note the `//?` in the location before the rewrite.

To ensure the rewrite only catches single slashes we ensure the location doesn't include a leading slash with `[^/]`